### ISSUE TITLE
QtWebEngine segfault after open 

### ISSUE THREAD

**Issue by peet1234** [qutebrowser/qutebrowser#3667]
crack everytime 

```
$ qutebrowser 
[0303/095332.574278:WARNING:stack_trace_posix.cc(648)] Failed to open file: /home/peet/#43318
  Error: No such file or directory
Received signal 11 SEGV_MAPERR 000000000081
#0 0x7f8870e32e76 <unknown>
#1 0x7f886fab088c <unknown>
#2 0x7f8870e331ec <unknown>
#3 0x7f88906a9dd0 <unknown>
#4 0x7f88719d1e4f <unknown>
#5 0x7f88719d2e1c <unknown>
#6 0x7f88719d5134 <unknown>
#7 0x7f88719f06a5 <unknown>
#8 0x7f88719f938f <unknown>
#9 0x7f887198b0e8 <unknown>
#10 0x7f887198b1be <unknown>
#11 0x7f88700e1173 <unknown>
#12 0x7f88700d66b2 <unknown>
#13 0x7f8870eb8c32 <unknown>
#14 0x7f8870e81b6a <unknown>
#15 0x7f8870e82008 <unknown>
#16 0x7f8870e81951 <unknown>
#17 0x7f8870ec1697 <unknown>
#18 0x7f8870e8713b <unknown>
#19 0x7f889069f08c start_thread
#20 0x7f88903d6e7f __GI___clone
  r8: 00000000000003dd  r9: 00005578ddb32993 r10: 0000000000000002 r11: 0000000000000001
 r12: 0000000000000002 r13: 0000000000000000 r14: 0000000000000001 r15: 00007f87ff7fd494
  di: 000000000000000b  si: 00005578ddb3298b  bp: 0000000000000009  bx: 00007f87ff7fd4d4
  dx: 0000000000000004  ax: 0000000000000001  cx: 00000000000004a7  sp: 00007f87ff7fd3a0
  ip: 00007f88719d1e4f efl: 0000000000010202 cgf: 002b000000000033 erf: 0000000000000006
 trp: 000000000000000e msk: 0000000000000000 cr2: 0000000000000081
[end of stack trace]
```

Run: qutebrowser --version

```
 $qutebrowser --version
[0303/115651.875025:WARNING:stack_trace_posix.cc(648)] Failed to open file: /home/peet/#4325619 (deleted)
  Error: No such file or directory
qutebrowser v1.1.2
Git commit: 
Backend: QtWebEngine (Chromium 61.0.3163.140)

CPython: 3.6.4
Qt: 5.10.1
PyQt: 5.10

sip: 4.19.7
colorama: no
pypeg2: 2.15
jinja2: 2.10
pygments: 2.2.0
yaml: 3.12
cssutils: no
attr: 17.4.0
PyQt5.QtWebEngineWidgets: yes
PyQt5.QtWebKitWidgets: yes
pdf.js: no
sqlite: 3.22.0
QtNetwork SSL: OpenSSL 1.1.0g  2 Nov 2017

Style: QFusionStyle
Platform: Linux-4.15.5-1-ARCH-x86_64-with-arch, 64bit
Linux distribution: Arch Linux (arch)
Frozen: False
Imported from /usr/lib/python3.6/site-packages/qutebrowser
Qt library executable path: /usr/lib/qt/libexec, data path: /usr/share/qt

Paths:
cache: /home/peet/.cache/qutebrowser
config: /home/peet/.config/qutebrowser
data: /home/peet/.local/share/qutebrowser
runtime: /run/user/1000/qutebrowser
system data: /usr/share/qutebrowser

Uptime: 0:00:00
```

---

*(edited by @The-Compiler to add "```" around code blocks)*

**Comment by The-Compiler** [comment]
I don't know what's happening there. Can you please [install my Qt packages with debugging symbols](https://github.com/qutebrowser/qutebrowser/blob/master/doc/stacktrace.asciidoc#archlinux) and see if the output changes? Also, can you please try to [get a stacktrace with gdb](https://github.com/qutebrowser/qutebrowser/blob/master/doc/stacktrace.asciidoc#getting-the-stack-trace)?

**Comment by peet1234** [comment]
lucky me, no problem now! didn't do anything.

**Comment by The-Compiler** [comment]
Weird! No idea what that was, then. Let me know if you see it again!

**Comment by McKean** [comment]
I'm getting the same error when closing the browser. Which then triggers a fallback so firefox is immediately opened. Any ideas?
```
dg-open http://google.com
23:04:23 INFO: Request to googleads.g.doubleclick.net blocked by host blocker.
/usr/bin/xdg-open: line 737: 23060 Segmentation fault      (core dumped) "$command_exec" "$@"
[0504/230425.764393:WARNING:stack_trace_posix.cc(648)] Failed to open file: /home/mckean/#17348876 (deleted)
  Error: No such file or directory
```
```
Version info

qutebrowser v1.0.4
Git commit: f5edd4941 (2017-12-30 18:33:15 +0100)
Backend: QtWebEngine (Chromium 61.0.3163.140)

CPython: 3.6.5
Qt: 5.10.1
PyQt: 5.10.1

sip: 4.19.8
colorama: 0.3.7
pypeg2: 2.15
jinja2: 2.10
pygments: 2.2.0
yaml: 3.12
cssutils: no
attr: 17.4.0
PyQt5.QtWebEngineWidgets: yes
PyQt5.QtWebKitWidgets: yes
pdf.js: 1.9.426 (/usr/share/pdf.js/build/pdf.js)
sqlite: 3.23.1
QtNetwork SSL: OpenSSL 1.1.0h  27 Mar 2018

Style: QStyleSheetStyle
Platform: Linux-4.14.36-1-MANJARO-x86_64-with-arch-Manjaro-Linux, 64bit
Linux distribution: Manjaro Linux (manjaro)
Frozen: False
Imported from /usr/lib/python3.6/site-packages/qutebrowser
Qt library executable path: /usr/lib/qt/libexec, data path: /usr/share/qt

Paths:
cache: /home/mckean/.cache/qutebrowser
config: /home/mckean/.config/qutebrowser
data: /home/mckean/.local/share/qutebrowser
runtime: /run/user/1000/qutebrowser
system data: /usr/share/qutebrowser

Uptime: 0:00:07
```

**Comment by The-Compiler** [comment]
@McKean That looks like an unrelated issue, and probably something which has already been fixed - you still seem to be running v1.0.4 which is from November 2017.
